app/templates/base.html

```diff
@@ -5,67 +5,170 @@
 
 <!DOCTYPE html>
 <html lang="en">
+  <head>
```

Comment:
Same as the other PR, lets not reformat everything and just use the team formatting that has been in use since the beginning